src/about-multiprotocol.html

```diff
@@ -0,0 +1,86 @@
+            spectrum (channel number 11-25). The standard supports multiple networks on
+            the same channel. Each network is identified by a PAN-ID (personal area network
+            identifier).
+        </span></p>
+    <p><span>
+            The wireless chip in Home Assistant SkyConnect supports all 16 channels. However,
+            it can only communicate on one single channel at a time. Due to this limitation
+            it is crucial that Zigbee and Thread run on the same channel for reliable
+            operation!
+
+            Thanks to the PAN-ID, the Zigbee and Thread networks are still separated and
+            therefore won't interfere with each other.
+        </span></p>
+    <h2>About bandwidth</h2>
+    <p><span>
+            IEEE 802.15.4 (so both Zigbee and Thread) support a transfer rate of up to
+            250&nbsp;kbit/s. For typical Zigbee and Thread networks this is sufficient.
+            However, in the multiprotocol setup Zigbee and Thread networks need to share
+            the same channel, and therefor share the 250&nbsp;kbit/s transfer rate. Furthermore,
+            if Wi-Fi or other 2.4&nbsp;GHz networks are running in a similar frequency range,
+            transfer rate might be further limited.
+        </span></p>
+    <p><span>
+            For large networks it is advisable to add a second wireless chip. This allows
+            to run Zigbee and Thread each on a dedicated wireless chip and on separate
+            channels.
+        </span></p>
+    <h2 id="resolve-channel-conflict">Resolve channel conflict</h2>
+    <p>You may get a notification stating that the OTBR (Open Thread border router) and ZHA share the same radio, but
+        use different channels. This mode of operation may cause an unreliable network connection. To resolve this
+        issue, you can try one of the following methods:
+    <ul>
+        <li><b>Option 1</b>: Reset the Thread network. This will make sure Thread uses the same channel as ZHA.
+            <ol>
+                <li>To reset the Thread network, select the <b>Reset border router</b> button. <img
+                        src="/images/multiprotocol-reset-thread-network.png" /></li>
+                <li>Re-pair your devices.</li>
+            </ol>
+        </li>
+        <li><b>Option 2</b>: Use two separate chips, one for each network. This works if you have a both Home Assistant
+            Yellow and SkyConnect.
+            <ol>
+                <li>On your Home Assistant Yellow, <a href="/guides/disable-multiprotocol/" target="_blank">disable
+                        multiprotocol support</a> and restore the Zigbee network.</li>
+                <li>Plug in a SkyConnect and setup a new Thread network there.</li>
+                <li>Go to <a class="my"
+                        href="https://my.home-assistant.io/redirect/addon/?addon=core_openthread_border_router"
```

Comment:
This link doesn't seem to work for me. I think it should be 

```suggestion
                        href="https://my.home-assistant.io/redirect/supervisor_addon/?addon=core_openthread_border_router"
```